language/fr/tech.lang

```diff
@@ -1,5 +1,121 @@
 <?php
 
-include("{$_EnginePath}language/en/tech.lang");
+$_Lang['Tech']                  = "Technologies";
+$_Lang['Requirements']          = "Prérequis";
+$_Lang['Metal']                 = "Métal";
+$_Lang['Crystal']               = "Cristal";
+$_Lang['Deuterium']             = "Deutérium";
+$_Lang['Energy']                = "Énergie";
+$_Lang['DarkEnergy']            = "Énergie noire";
+$_Lang['Message']               = "Messages";
+$_Lang['GoToShop']              = 'Boutique';
+$_Lang['level']                 = "Niveau";
+$_Lang['treeinfo']              = "[i]";
+
+$_Lang['Metal_rec']             = 'de métal';
+$_Lang['Crystal_rec']           = 'de cristal';
+$_Lang['Deuterium_rec']         = 'de deutérium';
+
+$_Lang['PrevPlanet']            = "Planète précédente";
+$_Lang['NextPlanet']            = "Planète suivante";
+
+$_Lang['type_mission'][1]       = "Attaquer";
+$_Lang['type_mission'][2]       = "Attaque groupée";
+$_Lang['type_mission'][3]       = "Transporter";
+$_Lang['type_mission'][4]       = "Stationner";
+$_Lang['type_mission'][5]       = "Défendre";
+$_Lang['type_mission'][6]       = "Espionner";
+$_Lang['type_mission'][7]       = "Coloniser";
+$_Lang['type_mission'][8]       = "Recycler";
+$_Lang['type_mission'][9]       = "Destruire";
+$_Lang['type_mission'][10]      = 'Attaque interplanétaire';
+$_Lang['type_mission'][11]      = 'Attaque groupée';
+$_Lang['type_mission'][15]      = "Expédition";
+$_Lang['type_mission_quickres'] = 'Transport de ressources';
+
+$_Lang['tech'] = array
+(
+    0 => "Bâtiments",
+    1 => "Mine de métal",
+    2 => "Mine de cristal",
+    3 => "Synthétiseur de deutérium",
+    4 => "Centrale électrique solaire",
+    12 => "Centre électrique de fusion",
+    14 => "Usine de robots",
+    15 => "Usine de nanites",
+    21 => "Chantier spatial",
+    22 => "Hangar de métal",
+    23 => "Hangar de cristal",
+    24 => "Réservoir de deutérium",
+    31 => "Centre de recherche",
+    33 => "Terraformeur",
+    34 => "Dépôt de ravitaillement",
+
+    40 => 'Bâtiments spéciaux',
+    41 => 'Base lunaire',
+    42 => 'Phalange de capteur',
+    43 => 'Porte de saut spatial',
+    44 => "Silo de missiles",
+    50 => "Porte quantique",
+
+    100 => 'Recherche',
+    106 => 'Espionnage',
```

Comment:
Looking at French Ogame wiki, it seems that almost all technologies have the `Technologie` prefix (in case of "technology X" kind of elements). Do you think it makes sense to add them here to make these translations "more complete", so to speak?

Reply:
I have noticed that OG uses "Technologie" before every tech element. In my opinion, (1) it is unnecessary since it is easy to understand, (2) it creates unpleasant repetitions, and (3) unnecessarily make titles longer. I would prefer to leave it like that. However, if you feel strongly about it, I can add them too.

Reply:
I think it would make a more cohesive translation experience with the prefix included. In addition to that, including the prefixes would make this translation similar to other games (original OG and other OG-like engines), so it would be more familiar to players migrating from other servers. I think that in this regard, we should strive to make the UI as much similar as possible, and I don't think that this repetition is necessarily that bad.

So yes, please include the prefixes.

Reply:
Ok. Done.